Python/ceval.c

```diff
@@ -6935,6 +6935,15 @@ PyEval_SetProfile(Py_tracefunc func, PyObject *arg)
     }
 }
 
+void
+PyEval_SetThreadProfile(PyThreadState *tstate, Py_tracefunc func, PyObject *arg)
+{
+    if (_PyEval_SetProfile(tstate, func, arg) < 0) {
+        /* Log _PySys_Audit() error */
+        _PyErr_WriteUnraisableMsg("in PyEval_SetProfile", NULL);
+    }
+}
```

Comment:
Is it possible to implement this API in https://github.com/python/pythoncapi-compat/ to check that it's possible to provide it to old Python versions?